api/src/services/sso-login.js

```diff
@@ -69,13 +63,10 @@ const oidcServerSConfig = async () => {
     token_endpoint,
   } = idServerConfig.serverMetadata();
 
-  const { client_id } = idServerConfig.clientMetadata();
-
   const connectionMetadata = `
   issuer: ${issuer}
   authorization_endpoint: ${authorization_endpoint}
   token_endpoint: ${token_endpoint}
-  client_id: ${client_id}
```

Comment:
I guess we don't really need to log this value since it should always just match the value from the `settings` doc.